node_agent/main.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Istio Authors
```

Comment:
Just filed #177. Let's move structure-related discussion over there. Once we have an agreement over there, I can refactor code regardless what is done in this PR.